CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [0.5.4] - 2022-03-17
+### Changed
+- Connection status listeners are all set to False by default.
+- Function set_max_velocity no longer changes the profile velocity, instead it changes velocity.
+
+### Added
+- Support to multi-slave Ethernet, EoE and CANopen connections.
+- Function connect_servo_ecat_interface_ip in communication.
+- Function get_ifname_from_interface_ip in communication.
+- Function get_current_loop_rate in configuration.
+- Function set_profiler in configuration.
+- Function set_max_profile_acceleration in configuration.
+- Function set_max_profile_deceleration in configuration.
+- Function set_max_profile_velocity in configuration.
+
+### Fixed
+- Commutation analysis feedback now returns the proper drive errors.
+
+### Deprecated 
+- Deprecated set_max_acceleration in configuration.
```

Comment:
In the set_max_acceleration should be added that the function is deprecated and what function should be used now.